spec/models/budget_spec.rb

```diff
@@ -8,6 +8,14 @@
     it { is_expected.to validate_uniqueness_of(:name) }
   end
 
+  context "associations" do
+    subject { build(:budget) }
```

Comment:
No need for the subject here. So you should be able to remove lines 12-13

Reply:
Done in sha: 1453820